scrapli/driver/base/base_driver.py

```diff
@@ -127,6 +127,11 @@ class that extends the driver, instead allowing the community platforms to simpl
             N/A
 
         """
+        if port is None:
+            port = 22
+            if "telnet" in transport:
```

Comment:
Add "asynctelnet" please!

Reply:
Good catch -- it looks like this is/was actually because of all the "higher layers" (generic driver, network driver, then the actual platforms -- i.e. iosxe/junos/etc.) had the port set to 22 -- so port was never `None`. Will push a fix for this in just a second 😁 then the `if "telnet" in transport` should work just fine for "telnet" or "asynctelnet" then we should be good to go!!